flutter-studio/src/io/flutter/module/FlutterModuleImporter.java

```diff
@@ -87,32 +97,49 @@ public void importModule() {
       return;
     }
 
+    editSettingsFile(settingsFile);
+    editBuildFile(buildFile);
+
+    // Setup a default run configuration for 'main.dart' (if it's not there already and the file exists).
+    PubRoot pubRoot = PubRoot.forDirectory(moduleRoot);
+    assert (pubRoot != null);
```

Comment:
Do we know whether this will fire? If so, we may instead want to check for null and handle it.

Reply:
The assert is just to keep IntelliJ happy. I don't think it can fire.